layout/js/map.js

```diff
@@ -64,6 +65,8 @@ function hovertext(text, evt){
 }
 
 function cursorPoint(evt){  
+  svg = document.querySelector("svg");
```

Comment:
@jiwalker-usgs I want to only call this once, but didn't see an init function, so it is called everytime here for now